### PR TITLE
dumb(stuff): PR demo comparison junk

### PR DESCRIPTION
**Pull request by joshsleeper**
## changes

- some dumb `stuff` :tm:
- edits after PR open

## ref

- github PR feature comparison
#123

### CODE REVIEW

File: src/pipx/colors.py [hunk 1]
```diff
@@ -1,22 +1,18 @@
-import sys
+import thegoodness
```
Comment (joshsleeper) [inline src/pipx/colors.py:1]: wow pretty sure this crap isn't real

```suggestion
import a_real_thing
```

File: src/pipx/colors.py [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
 from typing import Callable
 
 try:
     import colorama  # type: ignore
 except ImportError:  # Colorama is Windows only package
     colorama = None
 
-PRINT_COLOR = sys.stdout.isatty()
```
Comment (joshsleeper) [inline src/pipx/colors.py:9]: lol suck it `PRINT_COLOR`

File: src/pipx/colors.py [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
-if PRINT_COLOR and colorama:
-    colorama.init()
-
 
 class c:
     header = "\033[95m"
     blue = "\033[94m"
     green = "\033[92m"
     yellow = "\033[93m"
+    tracy = "\000[32x"
```
Comment (joshsleeper) [inline src/pipx/colors.py:15]:
```suggestion
    tracy4eva = "\000[32x"
```

**Review by joshsleeper — COMMENTED** [review]
just a few comments, because tracy5eva

File: src/pipx/colors.py (hunk 1 above)
Comment (tracyinspace) [reply src/pipx/colors.py:1]: This is a comment. The comment is from Tracy

File: src/pipx/colors.py [hunk 4]
```diff
@@ -1,22 +1,18 @@
-import sys
+import thegoodness
 from typing import Callable
```
Comment (tracyinspace) [inline src/pipx/colors.py:3]: blahblahblah

File: src/pipx/colors.py [hunk 5]
```diff
@@ -1,22 +1,18 @@
...
-PRINT_COLOR = sys.stdout.isatty()
-
-if PRINT_COLOR and colorama:
-    colorama.init()
-
 
 class c:
     header = "\033[95m"
     blue = "\033[94m"
     green = "\033[92m"
```
Comment (tracyinspace) [inline src/pipx/colors.py:13]:
```suggestion
    header = "\033[95m"
   # not blue = "#####"
    green = "\033[92m"
   # green = "\033[92m"
```

**Review by tracyinspace — CHANGES_REQUESTED** [review]
This is a comment

File: src/pipx/colors.py (hunk 1 above)
Comment (joshsleeper) [reply src/pipx/colors.py:1]: idc `thegoodness` is the best

**Pull request merged**